roles/ood_add_rstudio/templates/form.yml

```diff
@@ -0,0 +1,48 @@
+---
+cluster: "{{ cluster_name }}"
+form:
+  - version
+  - bc_account
+  - bc_num_hours
+  - node_type
+  - num_cores
+  - bc_email_on_started
+attributes:
+  num_cores:
+    widget: "number_field"
+    label: "Number of cores"
+    value: 1
+    help: |
+      Number of cores on node type (4 GB per core unless requesting whole
+      node). Leave blank if requesting full node.
+    min: 1
+    max: 28
+    step: 1
+  bc_account:
+    label: "Project"
+    help: "You can leave this blank if **not** in multiple projects."
+  node_type:
+    widget: select
+    label: "Node type"
+    help: |
+      - **any** - (*1-28 cores*) Use any available Owens node. This reduces the
```

Comment:
We probably want to replace these cluster name references to "Owens" with whatever the local cluster name is in our ansible config, eg. {{ cluster_name }}.

Reply:
Since we are maintaining this as a template we can remove the whole node type section or build a custom one for our site.  Looks like we aren't using this value in the submit script anyway.

Reply:
We can leave this for a feature enhancement though.